slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -130,6 +130,7 @@ ::grpc::Status SLAMServiceImpl::GetPointCloudMap(const GetMapRequest *request,
             // that the optimization is not ongoing and we can grab the newest
             // map
             pointcloud_has_points = ExtractPointCloudToString(pointcloud_map);
+            // pointcloud_has_points = GetLatestPointCloudMapString(pointcloud_map);
```

Comment:
Leaving the new implementation out of the GetMap() call, will be implemented in [RSDK-1074](https://viam.atlassian.net/browse/RSDK-1074?atlOrigin=eyJpIjoiZTM4ZGM3ZWExMmQ4NDlkY2FlOTE4Y2JjZDZiNDk0MzAiLCJwIjoiaiJ9)

Reply:
actually going to enable this for the PR, but will disable before merge

Reply:
based off tess's comment [here](https://github.com/viamrobotics/slam/pull/136#discussion_r1089047375) I will leave the new implementation in when complete

Reply:
Oh, I think I misunderstood. This PR is not yet adding the GetPointCloudMap gRPC function. Will that be in a follow-up PR for the same ticket?

I still think it probably makes sense to use the new implementation (GetLatestPointCloudMapString) for GetMap, since the old one (ExtractPointCloudToString) returned an incorrect pcd. We can also remove ExtractPointCloudToString.

Reply:
yeah that makes sense, will go ahead and swap it

Reply:
Sounds good. And will there be a follow-up PR for this ticket to add the GetPointCloudMap gRPC function?

Reply:
Yeah follow up PR will be the GetPointCloudMap function